using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md

```diff
@@ -0,0 +1,417 @@
+    ///
+    /// - Note: The returned XDM data does not contain an `eventType` for the Experience Event.
+    /// - Returns A dictionary containing XDM data for the propositon reference.
+    func generateReferenceXdm() -> [String: Any] {...}
+}
+```
+{% endtab %}
+{% endtabs %}
+
+The Edge `sendEvent` API can then be used to send this tracking XDM data along with any additional XDM and freeform data to the Experience Edge network. Additionally, an override dataset can also be specified for tracking data. For more details, see [Edge - sendEvent API](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension/edge-network-api-reference#sendevent).
+
+{% tabs %}
+{% tab title="iOS" %}
+#### Swift
+
+```swift
+// When a proposition is retrieved using getPropositions API or onUpdatePropositions API callback 
+// and the corresponding offer is displayed, invoke method on Offer object to get the XDM data.
+
+let displayInteractionXdm = offer.generateDisplayInteractionXdm() // Offer display tracking XDM
+let additionalData: [String: Any] = ["someDataKey": "someDataValue"]
+
+let experienceEvent = ExperienceEvent(xdm: displayInteractionXdm, data: additionalData)
+Edge.sendEvent(experienceEvent)
+```
+
+#### Objective-C
+
+```objc
+// When a proposition is retrieved using getPropositions API or onUpdatePropositions API callback 
+// and the corresponding offer is displayed, invoke method on Offer object to get the XDM data.
+
+NSDictionary* displayInteractionXdm = [offer generateDisplayInteractionXdm];
+NSDictionary* additionalData = @{@"someDataKey": @"someDataValue"};
+
+AEPExperienceEvent* experienceEvent = [[AEPExperienceEvent alloc] initWithXdm:displayInteractionXdm data:additionalData datasetIdentifier:nil];
+[AEPMobileEdge sendExperienceEvent:event completion:nil];
+```
+{% endtab %}
+{% endtabs %}
+
+## Configuration keys
+
+To update the SDK configuration programmatically, use the following information to change the Optimize extension configuration values. For more information, see [Programmatic updates to Configuration](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration/configuration-api-reference#updateconfiguration).
+
+| Key | Required | Description | Data Type |
+| :--- | :--- | :--- | :--- |
+| optimize.datasetId | No | Override dataset's Identifier which can be obtained from the Experience Platform UI. For more details see, [Datasets UI guide](https://experienceleague.adobe.com/docs/experience-platform/catalog/datasets/user-guide.html?lang=en) | String |
+
+{% hint style="info" %}
+If the override dataset is used for proposition tracking, make sure the corresponding schema definition contains the `Experience Event - Proposition Interaction` field group. For more information, see [Setup schemas & datasets](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/62a861aec745af2d8237c287656c68d8f8cdd5ed/getting-started/configure-schema-and-dataset.md).
```

Comment:
"For more information, see the [setup schemas and datasets] tutorial."

Reply:
Changed to [setup schemas and datasets guide]